tests/config/test_workers.py

```diff
@@ -94,6 +94,7 @@ def test_old_configs_appservice_worker(self) -> None:
                 # so that it doesn't raise an exception here.
                 # (This is not read by `_should_this_worker_perform_duty`.)
                 "notify_appservices": False,
+                "instance_map": {"main": {"host": "127.0.0.1", "port": 0}},
```

Comment:
Err, isn't port 0 reserved for binding? Presumably workers can't contact port 0?

Reply:
the communication is never used in these tests, this is just to make the validation happy. Maybe I should've chosen a less fake port number, but then if we accidentally *do* talk to it in the tests, we don't know if someone coincidentally has something bound to that port

Reply:
Oh we're just checking config parsing here?

Reply:
yup